ministryofjustice/laa-apply-for-criminal-legal-aid
# Implement passporting on ioj for u18 applicants

This adds a service to passport on ioj which includes calculating applicant age and adding passported type to ioj_passport. In turn, this results in the ioj section as being calculated as 'completed' and the badge updated on the task list For under 18 applicants, the step for entering ioj justification is replaced by an interruption card and saving and continuing from there takes the provider to the review page

## Link to relevant ticket
https://dsdmoj.atlassian.net/browse/CRIMAP-136

## Notes for reviewer
IojPassporter needs to be extendible for potential changes to age boundaries (16-18 may have separate rule to <16, we may want to use this to passport on ioj per offence in future)
I have added another DWP 'KNOWN' applicant to provide an under 18 input to manually test with


## How to manually test the feature
Go through the flow with user:
surname: 'POTTER' 
nino: 'NC010155A'
dob: '01-Jan-2007' 

After hearing_details step, you will be taken to the passported interruption card
Save and continue takes you to the review page. 
Navigate to the dashboard to see Ioj completed badge
